payjoin/src/send/mod.rs

```diff
@@ -172,6 +174,64 @@ pub struct Configuration {
 }
 
 impl Configuration {
+    // Calculate the recommended fee contribution for an Original PSBT.
+    //
+    // BIP 78 recommends contributing `originalPSBTFeeRate * vsize(sender_input_type)`.
+    // The minfeerate parameter is set if the contribution is available in change.
+    //
+    // This method fails if no recommendation can be made or if the PSBT is malformed.
+    pub fn recommended(
+        psbt: &Psbt,
+        mut payout_scripts: impl Iterator<Item = ScriptBuf>,
```

Comment:
If this is `IntoIterator` we dont' need it to be mut which makes for a nicer api